### PR TITLE
CSAF support short term hack

### PR DESCRIPTION
**Pull request by tschmidtb51**
This is the same as #93 but includes the short-term hack.

### CODE REVIEW

**Comment by joker314** [comment]
Thank you for submitting this. Sorry for taking so long to look at it, I will review it tomorrow and then merge it!

File: js/genform.js [hunk 1]
```diff
@@ -46,6 +46,9 @@ function generate(filename, field_array){
     // Converts camel case like 'abcDefGhi' into
     // the format 'Abc-Def-Ghi'
     function camelToHyphen(camelCaseWord) {
+        // introduce the special case for CSAF (short term solution)
+        if (camelCaseWord === "CSAF") {return "CSAF"}
```
Comment (joker314) [inline js/genform.js:50]: This looks good, just needs this change I think:
```suggestion
        if (camelCaseWord === "csaf") {return "CSAF"}
```

**Review by joker314 — CHANGES_REQUESTED** [review]

File: js/genform.js (hunk 1 above)
Comment (EdOverflow) [reply js/genform.js:50]: @joker314, the CSAF field is all caps in the regitry: https://www.iana.org/assignments/security-txt-fields/security-txt-fields.xhtml.

**Comment by tschmidtb51** [comment]
@joker314 Is that PR ok or do I need to change anything else?

**Comment by joker314** [comment]
Thank you! I've merged so it's now deployed at https://securitytxt.org

**Comment by tschmidtb51** [comment]
@joker314 Thank you so much!

**Pull request merged**